readthedocs/rtd_tests/tests/test_doc_builder.py

```diff
@@ -105,36 +98,17 @@ def test_create_conf_py(
             build_env=self.build_env,
             python_env=python_env,
         )
-        try:
+        with pytest.raises(ProjectConfigurationError):
             base_sphinx.append_conf()
```

Comment:
We should check that the msg error is the same as `ProjectConfigurationError.NOT_FOUND`